.travis.yml

```diff
@@ -0,0 +1,66 @@
+    - secure: "J61/KUCNGuWiZ6kiZDMjcKWFXGfnluE2+umVV8zgKX0MmGMv5ZyB1QIBNgQi72UUZSrvCUFh+lf77t7SgdHVM2hrsAITgFraFIl3Cyd3nfSoyoI2g7clhtar3rOovyg+hmScVDy96xeIgbT+DOzdEqKw5AEanQO9wZxI+X6jUcqOJsj8dTVPJwaNOfQ/416qQxGQXKQMyBRn9N6e/Im0UmpfifqYLvnWK5+ELhxuFNs7ZZd/nzQRgA97u9ytDzHC1SIIK2LPLgn64IaiBdau8jWqD2PiveYlKzLEAu7aiDkLF7KUcOkh8Q3UUbf0sJt3IO0TJRUe0hdvcbpHpgHHQlTdKUuTVijpN9spdWobuLIp1QUug+zv1vlFUvypt1wR80wmiv9xFPYHrOvV7hTksNReo6yDeIE2H/YSLQSSKJaWfvT9vXDxMj4aNb3UVWp0FTdmd4d0xFG94pxYImXI/ZyzFqPES33MDDSC8ffIJfyX9MmNsEl3e6PaeWrgulp4Nkhene25R+kLrosqw+cTPoc92Hctm6zndrbUVeiywBPbRjXiEWakJ2MDzAqzd4GP/1tuxIL1YY8MOpKw4SNV19Ke1ywy59iRAb8GY4nj66aQOgErOdkGla8s69KJrr221rh4PusisYDRrirMev4MS0b68snxNjcN0FEk+IBvlL4="
+  # Heroku
+    - herokuapp: rajrahmantodoapp
+    - secure: "u9PjQ6s5qpMRnQZVbHaQ3u/9JmI2ooMafJoLaFTeb7+b14x169/dxsvH+58iFLiqjQgyG3eoM539CwLOQ3PkeE3VFmSysQ4Sw2B9tqE9ee763Pb+mSnTceZ6l3/OAvJmb3ZO9P1p0GG5pTDBSl9PdXZQBYoN8Tcx6OrXR1pAMAmbme1BYuuvqeX+TQvj5gK7kdXK7IINKbRcIO6QmO4h/w4Bhh9q/p2iLsEqjoXWHVWpAaz4H/YWQTs5rT3/zaI1NzXf6vXYZikdr2c4Hlyhnv1q+P052r2hr9DKoI/qKQbY8Wd8/9dtmiCTpyzGz2MSh4wIxNYUcxQ8clzIasGmUfLTPJiL0Rbu9soly5/mAV73na2g6vRO12Hng4MlR4C1pfAk541Vx1pll5+9/HSu3Bw4tyngOc1y0knn6H7HWPz3CfuPAz45PaZCn9QI+Zt0/5jlBeXC5Ez6kV2n4Sywt3hDevZP2lbq4Q0dtQGd6au0u+Os3TMq+jTg13Cj/u9Zbyoce5ywZA1W8EXqtSNu7MCvSD0bcbIn4HOFA85m5Qjbblg5NkxbW+xA3lrH/BhkW/B1d1ifVxTV4reuaNlR9q+KYEI8I9ZvOxnZgfenl5IjplXx4ayTVXny9JaM0vJTjQ2XPzxYlkwuFdAIpN3gq8F5FmlajDsULfpDTCBhxxY="
+    - secure: "gOWWEMWMUnNWJmwH25wEzjcbOMhA6E/WHyNoI1QclNZFrHtUhhZXYO53lTWtAgvRZf9D0ouPmwAcE7Hgbjh/ql3ERWpsU8W0XiolCopd5zMfYp4gpxiQVdiYTyEW8WYv8mrtwQ/lHzWSij58MAsgBDML/IYdgHHvl7Ty8b2tGE3dUd+wrLsE8nZ3XohyUELnqfNAC5+qPOv3vWSL3cAVOSSAkWsrsowbpnbprpNmV18HO0dZZno3cGj3Sj9KOUTE6ceS1TSyz9oyzyZ3zp63IUkISGS6hOcMmQEU1toixO80tQG447mkRYAZlsXo6z3rjYNNvxVZsmKFxreYeBZGbOJeiW7fI8ZBin2fhuRoERYP84psB1IFEvOuBGyrHZiScZgmFPF0r75pp8we7w/3fBQq35luCnL5hJQ13V28JFDyHYUPAm5+alzUIA2hOBjYBtGVhSSevLlpQrLUdGmxqZFWumo2qXXWxsabsbwWG4oK3ixCJP1ET0rkls5BmLr1MjWLJN30miekP54O/gk23yWH0XviAbs8MOUY+Mel6TiwEF2N1X7UdjfMSrcnKknv78dfvoAFByqlT54oiZv6yzpb7cyFuQfqRWBiMDKulu6u819F5UQzE7Z+rVOQnXglPkrwuJu8ky8Lrsdd2Lfrbc6Zaz3w5+Tcixux1jbVvhQ="
+
+services:
+  - docker
+
+before_script:
+  - curl https://cli-assets.heroku.com/install.sh | sh
+  - docker login --username $docker_login --password $docker_password
```

Comment:
We should ideally action this step using the `--password-stdin` flag rather than passing the password explicitly, to avoid leaking those credentials to shell logs. [See the docs](https://docs.docker.com/engine/reference/commandline/login/) for more info, example of how that might look below:
```
echo $PASSWORD | docker login -u $USER --password-stdin
```